libevmasm/AssemblyItem.cpp

```diff
@@ -327,6 +380,30 @@ std::string AssemblyItem::toAssemblyText(Assembly const& _assembly) const
 	case VerbatimBytecode:
 		text = std::string("verbatimbytecode_") + util::toHex(std::get<2>(*m_verbatimBytecode));
 		break;
+	case RelativeJump:
+		text = "rjump(" + std::string("tag_") + std::to_string(static_cast<size_t>(data())) + ")";
+		break;
+	case ConditionalRelativeJump:
+		text = "rjumpi(" + std::string("tag_") + std::to_string(static_cast<size_t>(data())) + ")";
+		break;
+	case CallF:
+		text = "callf(" +  std::to_string(static_cast<size_t>(data())) + ")";
+		break;
+	case JumpF:
+		text = "jumpf(" +  std::to_string(static_cast<size_t>(data())) + ")";
+		break;
+	case RetF:
+		text = "retf";
+		break;
+	case EofCreate:
+		text = "eofcreate(" +  std::to_string(static_cast<size_t>(data())) + ")";
+		break;
+	case ReturnContract:
+		text = "returcontract(" +  std::to_string(static_cast<size_t>(data())) + ")";
```

Comment:
typo in assembly text of `RETURNCONTRACT`
```suggestion
		text = "returncontract(" +  std::to_string(static_cast<size_t>(data())) + ")";
```